addons/purchase/views/purchase_views.xml

```diff
@@ -752,7 +752,7 @@
     
     <!-- Dashboard action buttons -->
     <!-- Dashboard action buttons: End in List view -->
-    <record id="purchase_action_dashboard_list" model="ir.actions.act_window">
+    <!-- <record id="purchase_action_dashboard_list" model="ir.actions.act_window">
```

Comment:
Unfortunately, It is used in `purchase_dashboard.js` (`action_name: $action.attr('name')+"_list"`). Same than for the kanban view.

Reply:
That's probably why dynamically building names is a bad idea :D

Reply:
There will be no more soon, https://github.com/odoo/odoo/pull/56211